docs/open-source-project/release-notes/tizen-5-5-m1.md

```diff
@@ -0,0 +1,631 @@
+
+
+## Service framework
+
+### New and changed features
+
+- Account -Manager
+  - Added feature to query account information even after account deletion.
+
+### Fixes
+
+- Account-Manager
+  - Removed unwanted D-Bus API calls.
+  - Fixed Memory Leaks.
+  - Fix to remove provider information from global account db.
+- Sync-Manager
+  - Enabled Additional D-Bus error handling.
+  - Handled potential defects.
+- Calendar-service
+  - Supported calendar DB purge when Device is reset.
+  - Fixed potential defects.
+- Email-service
+  - Fixed Potential defects.
+- Message Service
+  - Fixed Potential defects.
+- Location Service
+  - Handled potential defects.
+  - Replaced deprecated capi-network-wifi dependency.
+- Sensord
+  - Fixed potential defects.
+  - Fixed timestamp data type issue.
+- Phonenumber-Utils
+  - Fixed Potential defects.
+
+
+## Web framework
+
+### New and changed features
+
+- Web Engine
+  - Web Engine Upgrade Framework has been added.
+    - chromium-efl package can be upgraded with TPK, not whole image upgrade.
+    - chromium-efl-squashfs package has been added to provide the preload web engine as a squashfs image file, not regular file.
+    - chromium-efl squashfs image file is compressed by about 60%.
+    - chromium-efl, chromium-efl-update systemd services have been added for squashfs image mount.
+    - chromium-efl-install systemd service has been added for tpk installation.
+  - C# API
+    - WebView
+      - API set to resume/suspend has been added.
+      - API set to handle scale, scroll, zoom, and orientation has been added.
+      - API to load URL with given request date has been added.
+      - API to find text in page has been added.
+      - Events to handle policy decision have been added.
+      - Event and delegate to handle context menu have been added.
+    - BackForwardList
+      - API set to handle back/forward list has been added.
+    - Context
+      - API to clear resource cache has been added.
+    - ContextMenu
+      - API set to handle context menu has been added.
+    - PolicyDecision
+      - API set to handle navigation policy decision has been added.
+      - API set to handle response policy decision has been added.
+      - API set to handle new window policy decision has been added.
+    - Settings
+      - API to set whether window can be opened by script has been added.
+  - Autofill
+    - Autofill feature that works with autofill framework has been added.
+    - This feature automatically fills out a login form with appropriate data from autofill framework.
+    - Supports default popups for authentication and accounts.
+  - CSS Scroll Snap Feature has been added.
+  - IME support for NUI has been added.
+- Web Runtime
+  - WRTjs (the new type of JavaScript-based WRT Framework) has been added.
+  - WRTjs is based on the open source Electron project.
+  - Providing compatibility with the legacy Tizen WRT Apps.
+  - Providing compatibility with the existing Tizen API set.
```

Comment:
Suggest rephrasing to:

Compatibility with the existing Tizen API set. has been provided.

Reply:
will accept